dynamic_programming/wildcard_string_matching.cpp

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
For how to document the first comment block, please see the [typical structure](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program) of a program.